dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -76,28 +76,28 @@ useAuthenticationDataForUnprotectedResource=Use authenticated identity for an un
-allowAuthenticationFailOverToAuthMethod.desc= Specifies the authentication fail over method that will be used when the certificate authentication fails. Valid values are BASIC and FORM.
+allowAuthenticationFailOverToAuthMethod=Fail over method when certificate login fails
+allowAuthenticationFailOverToAuthMethod.desc=Specifies the authentication fail over method that will be used when the certificate authentication fails or if the certificate is missing. Valid values are BASIC, FORM and APP_DEFINED. APP_DEFINED is only valid when overrideHttpAuthMethod attribute is set to CLIENT_CERT. When APP_DEFINED is set, the authentication method which is  configured in the application is used.
 
 includePathInWASReqURL=Set the path parameter for the WASReqURL cookie
 includePathInWASReqURL.desc=Setting the Path parameter can allow the client/browser to manage multiple WASReqURL cookies during multiple concurrent logins on the same user agent. The equivalent custom property in the full application server profile is com.ibm.websphere.security.setContextRootForFormLogin.
 
 trackLoggedOutSSOCookies=Track logged out LTPA single signon tokens
 trackLoggedOutSSOCookies.desc=Specifies whether to track LTPA single signon tokens that are logged out on a server so that it can not be reused on the same server.
 
-overrideHttpAuthenticationMechanism=Name of HttpAuthenticationMechanism to override
-overrideHttpAuthenticationMechanism.desc=Specifies the HttpAuthenticationMechanism for overriding an application-defined HttpAuthenticationMechanism. The Java EE 8 Security specification defines the HttpAuthenticationMechanism. The acceptable value is BASIC or FORM. When FORM is used, specify the loginFormURL and loginErrorURL properties.
+overrideHttpAuthMethod=Name of authentication method to override
+overrideHttpAuthMethod.desc=Specifies the authentication method to be used for all applications. This specified value overrides any application defined authentication method. The acceptable value is BASIC, FORM or CLIENT_CERT. When FORM is used, loginFormURL and loginErrorURL attributes are required to be set.
 
 contextRootForFormAuthenticationMechanism=Context root for the Java EE 8 login form
-contextRootForFormAuthenticationMechanism.desc=Specifies the context root of a form login page, which is specified by the loginFormURL property. If this value is not set, the first path element of the loginFormURL property is used as a context root. This value is applicable when overrideHttpAuthenticationMechanism is set to FORM.
+contextRootForFormAuthenticationMechanism.desc=Specifies the context root of a form login page, which is specified by the loginFormURL property. If this value is not set, the first path element of the loginFormURL property is used as a context root. This value is applicable when overrideHttpAuthMethod is set to FORM, or if overrideHttpAuthMethodis set to CLIENT_CERT and allowAuthenticationFailOverToAuthMethod is set to FORM.
```

Comment:
Change: 
if overrideHttpAuthMethodis set to CLIENT_CERT
To:
if overrideHttpAuthMethodis is set to CLIENT_CERT

Reply:
Fixed. Thank you.